cernopendata/modules/fixtures/data/records/jade-data.json

```diff
@@ -0,0 +1,2087 @@
+[
+  {
+    "abstract": {
+      "description": "<p>All data from the JADE detector at the PETRA accelerator taken from 1979 to 1986 with e+e- collision energies ranging from 12 to 46.7 GeV. JADE was one of the detectors responsible for the discovery of the gluon.</p><p>This data was recovered in recent years as it had to be moved due to space problems and was migrated multiple times over the years as more compact data storage devices became available.</p>",
+      "links": [
+        {
+          "description": "Article about history, achievements and revival of the JADE experiment",
+          "url": "https://arxiv.org/pdf/2208.11076"
+        }
+      ]
+    },
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "26050"
+    },
+    "collections": [
+      "JADE-Primary-Datasets"
+    ],
+    "collision_information": {
+      "energy": "12GeV",
+      "type": "e+e-"
+    },
+    "date_created": [
+      "1986"
+    ],
+    "date_published": "2024",
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 0,
```

Comment:
Why the change to the "0" as the number of events? The previous number 7017835 wasn't right? Will the number "0" be updated before the merge?